### PR TITLE
ISSD-2814 [Testray] Add logic to not add filter to URL when modal

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

94 files changed, 3178 insertions(+), 2520 deletions(-)

This PR is too huge for one to review :broken_heart: 

Batman

**Comment by rafaelfs98** [comment]
ci:test:sf

**Comment by rafaelfs98** [comment]
Resolved @kevenleone  @joseabelenda

**Comment by rafaelfs98** [comment]
ci:test:sf

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx [hunk 1]
```diff
@@ -30,8 +34,33 @@ const ManagementToolbarResultsBar: React.FC<ManagementToolbarResultsBarProps> =
 		dispatch({payload: null, type: ListViewTypes.SET_CLEAR});
 	};
 
+	const handleRemoveItemFromFilter = (itemToRemove: string) => {
+		const searchParams = new URLSearchParams(location.search);
+
+		const filtro = searchParams.get('filter');
```
Comment (kevenleone) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx:40]:
```suggestion
		const filter = searchParams.get('filter');
```

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx [hunk 2]
```diff
@@ -30,8 +34,33 @@ const ManagementToolbarResultsBar: React.FC<ManagementToolbarResultsBarProps> =
...
+
+		if (filtro) {
+			const filtroJSON = JSON.parse(decodeURIComponent(filtro));
+
+			delete filtroJSON[itemToRemove];
+
+			if (!Object.keys(filtroJSON).length) {
```
Comment (kevenleone) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx:47]: Pay attention the word needs to be in English

```suggestion
		if (filter) {
			const filterJSON = JSON.parse(decodeURIComponent(filter));

			delete filterJSON[itemToRemove];

			if (!Object.keys(filterJSON).length) {
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
LGTM 🔥 

See my last comment please

**Comment by kevenleone** [comment]
ci:forward

**Comment by rafaelfs98** [comment]
ci:stop

**Comment by rafaelfs98** [comment]
ci:stop

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/147026

**Pull request closed without merging**